File: conditions/drinks-allowed.py
def should_serve_drinks(age, on_break):
	return age >= 18 and not on_break

should_serve_drinks(17, True) # False
should_serve_drinks(30, True) # False
should_serve_drinks(24, False) # True
should_serve_drinks(18, False) # True
should_serve_drinks(3, False) # False
should_serve_drinks(18, True) # False
should_serve_drinks(17, False) # False

File: conditions/drinks-allowed.cpp
bool shouldServeDrinks(int age, bool onBreak) {
	return age > 17 && !onBreak;
}

Describe(shouldServeDrinks_){
It(test1){Assert::That(shouldServeDrinks(17, true), Equals(false));}
It(test2){Assert::That(shouldServeDrinks(30, true), Equals(false));}
It(test3){Assert::That(shouldServeDrinks(24, false), Equals(true));}
It(test4){Assert::That(shouldServeDrinks(18, false), Equals(true));}
It(test5){Assert::That(shouldServeDrinks(3, false), Equals(false));}
};